bin/i18n/i18n_script_utils.rb

```diff
@@ -289,38 +281,6 @@ def self.fix_yml_file(filepath)
     File.write(filepath, yml_data)
   end
 
-  # Return true iff the specified file in the specified locale had changes
-  # since the last successful sync-out.
-  #
-  # @param locale [String] the locale code to check. This can be either the
-  #  four-letter code used internally (ie, "es-ES", "es-MX", "it-IT", etc), OR
-  #  the two-letter code used by crowdin, for those languages for which we have
-  #  only a single variation ("it", "de", etc).
-  #
-  # @param file [String] the path to the file to check. Note that this should be
-  #  the relative path of the file as it exists within the locale directory; ie
-  #  "/dashboard/base.yml", "/blockly-mooc/maze.json",
-  #  "/course_content/2018/coursea-2018.json", etc.
-  def self.file_changed?(locale, file)
```

Comment:
out of curiousity, what is this replaced with?

Reply:
The sync-out process was changed (check the description), so we don't need this anymore